src/components/plots/rectanglePlot.ts

```diff
@@ -97,16 +130,23 @@ export module Plots {
       return this;
     }
 
-    public y2(): AccessorScaleBinding<Y, number>;
-    public y2(y2: number | Accessor<number>): Plots.Rectangle<X, Y>;
-    public y2(y2: Y | Accessor<Y>, scale: Scale<Y, number>): Plots.Rectangle<X, Y>;
-    public y2(y2?: number | Accessor<number> | Y | Accessor<Y>, scale?: Scale<Y, number>): any {
-      if (y2 == null) {
-        return this._propertyBindings.get(Rectangle._Y2_KEY);
+    private _bandWidth(scale: Scale<any, number>) {
```

Comment:
Let's not name this `_bandWidth()` but instead like `_rectangleWidth()`?  Band width is describing something similar to the amount of space a discrete object has in a designated area (on a "Scales.Category" you know your entire domain beforehand so you know how much space each entry gets)